tests/sqlalchemy_test_suite/__init__.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All right reserved.
```

Comment:
You should update all of the copyrights. You could copy the pre-commit hook that we use in the Python connector to make migrating these simple in the future

Reply:
I've put the changes in another PR: https://github.com/snowflakedb/snowflake-sqlalchemy/pull/306